### ISSUE TITLE
Rhode Island county maps not shading in

### ISSUE THREAD

**Issue by alykat** [nprapps/elections24-primaries#162]
For Rhode Island (Dem and GOP), we have county-level results, but the map’s not coloring in.
https://apps.npr.org/primary-election-results-2024/states/RI.html#date=4%2F2%2F2024&office=P&party=Dem&counties=true&state
https://apps.npr.org/primary-election-results-2024/states/RI.html#date=4%2F2%2F2024&office=P&party=GOP&counties=true&state

Results appear in the tooltip, however, and I can click a county to show the results table:
<img width="995" alt="image" src="https://github.com/nprapps/elections24-primaries/assets/491318/686226d3-177e-41d9-a2e0-50f3a504b43a">

Seeing this error in console:
<img width="860" alt="image" src="https://github.com/nprapps/elections24-primaries/assets/491318/3052f4bf-9515-4a9f-9ae2-3a03d04cdd78">

And there is an "undefined" county in the dropdown:
<img width="313" alt="image" src="https://github.com/nprapps/elections24-primaries/assets/491318/52444f59-2ea6-442b-90ee-918b68ca83cc">

**Comment by abidishajia** [comment]
@hilaryfung and I paired on this ticket.

1. We aren't able to click on the dropdown on localhost. 
2. On prod, it's kind of random. I was able to select from the dropdown for republican, but after reloading, both of the dropdowns don't work. They don't work for Hilary either.
3. Fips seem to be different. The one from AP starts from **40** and if we inspect the county map svg id start from **44**.
